src/modules/systemlib/CMakeLists.txt

```diff
@@ -31,7 +31,13 @@
 #
 ############################################################################
 
+px4_generate_parameters(OUT param_files)
```

Comment:
Can we do this in a separate param lib so adopters like APM can just not build that one library?